what_is.md

```diff
@@ -273,32 +306,61 @@ The trees reflect, for example, the origin and age of alleles under
 selection, the spatial structure of populations, and the effects
 of hybridization and admixture in the past.
 
-```{todo}
-Insert illustration of the above, perhaps a demesdraw plot, linking to the tree sequence
-in the next code box.
-```
+The tree sequence in the tutorial was actually generated using a model of population
+splits and expansions as shown in the following schematic, plotted using the fantastic
+[DemesDraw](https://pypi.org/project/demesdraw/) package. Our 10 genomes were sampled
+from modern day populations A (a constant-size population) and B (a recently expanding
```

Comment:
They're ``Pop_A`` and ``Pop_B`` in the model - I think it would be better to call them ``A`` and ``B``. The "Pop_" prefix is a bit like calling a variable ``my_var_a``

Reply:
I started with that, but wondered if without a caption it would be obvious that the tubes represented populations. But maybe if it's been mentioned that they are "populations" just above, that's OK. We've got to remember that this is the first page people will see, and they might not be used to this sort of viz (which is why I said "constant size" and "recently expanding", so hopefully it's clear)

Reply:
I think it's solved by naming the ancestral population "Ancestral_population", which (although it's a bit long) makes it very clear what the diagram is. I've pushed that and deleted "fantastic"